spec/menu_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'menu'
+describe Menu do
+  it 'creates instance' do
+    menu = Menu.new
+  expect(menu).to be_instance_of(Menu)
+  end
+
+   it 'shows dishes and prices' do
+     menu = Menu.new
+   expect(menu.food_list).to eq(:chips => 1.5, :chicken => 5.25)
+   end
+end
```

Comment:
I think this section creates a clear and simple description of what 'Menu' should do.